cmd/up.go

```diff
@@ -266,11 +269,16 @@ func (up *UpContext) Execute(isRetry bool) error {
 		return err
 	}
 
-	if err := up.Sy.Run(up.Context, up.WG); err != nil {
-		return err
+	syncGUIAddress := ""
+	if runSync {
+		if err := up.Sy.Run(up.Context, up.WG); err != nil {
+			return err
+		}
+		syncGUIAddress = up.Sy.GUIAddress
+
 	}
 
-	err = storage.Insert(up.Context, up.WG, up.Namespace, up.Dev, up.Sy.GUIAddress, up.Pod.Name)
+	err = storage.Insert(up.Context, up.WG, up.Namespace, up.Dev, syncGUIAddress, up.Pod.Name)
```

Comment:
could you just pass up.sy.GUIAddress here?

Reply:
If the UI address is empty, this could break how `cnd list` detects stale services. Can you check that scenario?